### PR TITLE
[fix] Fix garbled of table or column comments contain Chinese characters(#401)

### LINKED ISSUE #401: [Bug] When table or column comments contain Chinese characters would cause garbled.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

flink-doris-connector-1.18:1.6.0

### What's Wrong?

When table or column comments contain Chinese characters would cause garbled.

image

image

### What You Expected?

dispay the correct content

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by qg-lin**
# Proposed changes

Issue Number: close #401 

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

@JNSimba Could you please help me to review this PR?

### CODE REVIEW

File: flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java [hunk 1]
```diff
@@ -50,9 +51,11 @@ public DorisOptions(
             String password,
             String tableIdentifier,
             String jdbcUrl,
-            boolean redirect) {
+            boolean redirect,
+            String charsetEncoding) {
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java:55]: It seems that this parameter is only used by SchemaChangeManager. It is recommended to add it to the constructor of SchemaChangeManager separately.

Comment (qg-lin) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java:55]: Maybe other will use this parameter, such as flink cdc doris pipeline

Comment (qg-lin) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java:55]: This parameter is only used by SchemaChangeManager currently, but I'm not sure if other modules have the same problem, so wouldn't it be better to expose it here?

Comment (qg-lin) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java:55]: BTW, what time about next release

Comment (JNSimba) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java:55]: Doris currently only supports utf8 encoding～

Comment (qg-lin) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java:55]: Fine

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**